### ISSUE TITLE
can't achieve the same performance

### ISSUE THREAD

**Issue by ardellelee** [lawlietAi/Reproduction-Of-CRCNN-Via-Pytorch#1]
Hi,
I read the same paper about CRCNN and find your implementation. I noticed that you achieved f1 score of 70% on test data at ~50 epochs.  However, I'm running your code but can't achieve a similar performance:(

Could you please share some details like:
    - if you pre-trained the embeddings yourself with word2vec model on SemEval dataset, or use other embeddings? (I'm using pre-trained glove.6B.50d)
    - are the hyper-parameters in your code the final ones you use to achieve 70% f1 score? 
    - can you remember the version of PyTorch in use? (I'm using ver 0.3 released in Dec 2017)

Thank you.

**Comment by lawlietAi** [comment]
is there any progress？

**Comment by ardellelee** [comment]
> is there any progress？

Hi, thanks for the prompt reply. 

My current performance with 300d embedding is: 
`epoch: 23	 loss: 0.413812	 train_f1: 99.750	 test_f1: 70.741`

Btw I use adam for the result. Meanwhile I'm doing some fine-tuning with sgd, but haven't come out with a better result since it's slow.

**Comment by lawlietAi** [comment]
yeah, these times I also be limited by the performance of my computer. Adam has a faster convergence ability, but more fluctuation.

**Comment by lawlietAi** [comment]
yeah, these times I was also limited by the performance of my computer. Adam has a faster convergence ability, but more fluctuation.
